ocaml/default.nix

```diff
@@ -3,7 +3,6 @@
 oself: osuper:
```

Comment:
Sorry about the huge diff here, I installed a nix formatter and forgot to check the diff here, I can fix that.

Reply:
please revert this diff. happy to merge a PR that formats the whole thing, separately, if you want.